pkg/nfd-worker/nfd-worker.go

```diff
@@ -48,7 +48,7 @@ import (
 	"sigs.k8s.io/node-feature-discovery/source/local"
 	"sigs.k8s.io/node-feature-discovery/source/memory"
 	"sigs.k8s.io/node-feature-discovery/source/network"
-	"sigs.k8s.io/node-feature-discovery/source/panic_fake"
+	panicfake "sigs.k8s.io/node-feature-discovery/source/panic_fake"
```

Comment:
nit: this change is unrelated and unnecessary

Reply:
this was my IDE to blame, sorry will fix

Reply:
this import name convention is being recommended by both VIM-GO and VSCODE go plugins, should we keep it?

Reply:
#458 removes that all together (if we agree to merge that) so I would not change this in order to avoid unnecessary rebases.

Reply:
Ah, problem solved. #458 was merged 😄

Reply:
Although now you need to rebase 😬

Reply:
hahahaha ok rebasing now

Reply:
done, now working on the commit message